.travis.yml

```diff
@@ -27,65 +27,40 @@ after_success:
  - codecov
 
 _helpers:
-- &_mainstream_python_base
+  # anchor nodes to be included elsewhere.
+  # names that start with _ are only used in other helpers
+- &mainstream_python_base
   python: *mainstream_python
 - &_reset_steps
   env: []
   before_install: skip
   install: skip
   script: skip
-  after_success: []
-- &_lint_base
-  stage: &doc_stage_name docs, linting and pre-test checks
-  <<: *_mainstream_python_base
+
+- &lint_base
+  <<: *mainstream_python_base
   <<: *_reset_steps
+  stage: &doc_stage_name docs, linting and pre-test checks
+  env:
+    AIOHTTP_NO_EXTENSIONS: "1"
   install:
   - *upgrade_python_toolset
-  - pip install -U -r requirements/ci.txt
-- &_doc_base
-  <<: *_lint_base
-  install:
-  - *upgrade_python_toolset
-  - pip install -U -r requirements/doc.txt -r requirements/doc-spelling.txt
-  after_failure: cat docs/_build/spelling/output.txt
-  addons:
-    apt:
-      packages:
-      - libenchant-dev
-- &osx_python_base
-  stage: &stage_test_osx_name test under OS X (last chance to fail before deploy available)
-  os: osx
-  language: generic
-  python: *pypy3
-  env:
-  - &env_pypy3 PYTHON_VERSION=pypy3.6-7.1.1
-  - &env_pyenv PYENV_ROOT="$HOME/.pyenv"
-  - &env_path PATH="$PYENV_ROOT/bin:$PATH"
-  before_install:
-  - brew update
-  - brew install pyenv || brew upgrade pyenv
-  - &ensure_pyenv_preloaded |
-    eval "$(pyenv init -)"
-    eval "$(pyenv virtualenv-init -)"
-  - &install_python pyenv install --skip-existing --keep --verbose "$PYTHON_VERSION"
-  - &switch_python pyenv shell "$PYTHON_VERSION"
-  - &python_version python --version
-  before_cache:
-  - brew --cache
-- &generic_deploy_base
-  stage: &deploy_stage_name deploy (PYPI upload itself runs only for tagged commits)
-  <<: *_mainstream_python_base
-  deploy: &deploy_step
+  - *ci_requirements
+  before_script: skip
+
+- &_generic_deploy_base
+  <<: *_reset_steps
+  deploy: &_deploy_defaults
     provider: pypi
     # `skip_cleanup: true` is required to preserve binary wheels, built
     # inside of manylinux1 docker container during `script` step above.
     skip_cleanup: true
-    # `skip-existing: true` is required to skip uploading dists, already
+    # `skip_existing: true` is required to skip uploading dists, already
     # present in PYPI instead of failing the whole process.
     # This happens when other CI (AppVeyor etc.) has already uploaded
     # the very same dist (usually sdist).
-    skip-existing: true
-    user: aio-libs-bot
+    skip_existing: true
```

Comment:
Keep dashes.

Reply:
This is a warning in the travis config view. More-over, *every other key in the config uses underscores*, using dashes here is inconsistent and confusing. This isn't the `twine` command-line, this is the `deploy` configuration.

Reply:
I've found that using kebab-case for identifiers reads better in YAML. That's why I prefer dashed options where possible. These keys correspond to CLI options of a tool called Dpl (by Travis). I even contributed one provider there, I do know how it works. So it's only a matter of documentation.